eng/pipelines/linux.yml

```diff
@@ -23,16 +23,25 @@ jobs:
       displayName: Linux
       strategy:
         matrix:
+          # Run RedHat6 in release mode on CI to cover Release configuration differences
+          RedHat6_x64_Release:
+            _BuildConfig: Release
+            _architecture: x64
+            _framework: netcoreapp
+            _buildExtraArguments: /p:RuntimeOS=rhel.6 /p:PortableBuild=false
+            _dockerContainer: rhel6_container
+            _helixQueues: $(redhatHelixQueue)
+            _publishTests: ${{ parameters.fullMatrix }}
```

Comment:
This should be set to false, no? We didn't use to publish test artifacts of RedHat on official builds. I don't expect coreclr/mono running corefx tests on RedHat.

Reply:
> I don't expect coreclr/mono running corefx tests on RedHat.

My impression was that at least Mono was going to expand their test platform matrix. @BruceForstall any plans for RedHat?

Reply:
Currently, corefx-in-coreclr-repo testing is planned to test a minimum of platforms, since we expect to get almost all the coverage benefits from any individual Linux (probably will just do Ubuntu 1804), and testing on additional platforms adds cost and very little benefit.